tests/strictEqual.test.js

```diff
@@ -0,0 +1,32 @@
+const describe = require('mocha').describe
+const it = require('mocha').it
+const expect = require('chai').expect
+const sEqual = require('../strictEqual')
+
+describe('StrictEqual', () => {
+	describe('sEqual', () => {
```

Comment:
You should have four tests, one for each scenario

1. Same type, same value (ex. 1, 1)
2. Same type, different value (ex. 1, 3)
3. Different type, same value (ex. 1, '1')
4. Different type, different value (ex. 1, 'a')